### ISSUE TITLE
How long the training should take?

### ISSUE THREAD

**Issue by Joshua-Yu** [Octavian-ai/gqa-node-properties#1]
Hi David, 

I have been training the model for 24 hours by now(using default settings) and it's been running 7.6+ millions steps. The global_step per second is about 75. I am running it in a Docker CentOS container with 4GB RAM and 2 CPU cores (i7 7500 2.7G). 

Do you think it is the expected behavior? How many steps it would do?

Cheers

Joshua

**Comment by davidhughhenrymack** [comment]
Hello!

It should only need a few minutes of training. What’s your test accuracy at?

**Comment by Joshua-Yu** [comment]
I'd think so too. I didn't change any settings from the repository, and used this to run: 

$ python3.6 -m macgraph.train

The last lines on screen before I terminated the execution were: 
... ...
INFO:tensorflow:global_step/sec: 81.7011
INFO:tensorflow:loss = 0.0, step = 10511101 (1.224 sec)
INFO:tensorflow:loss = 0.0, step = 10511101 (1.224 sec)

Where do I set up the accuracy? 

Cheers

Joshua

**Comment by davidhughhenrymack** [comment]
Loss = 0.0 implies that the accuracy is probably really high.

You can see the accuracy by running `pipenv run tensorboard —logdir
./model` or similar

Enjoy :)


On March 3, 2019 at 3:31:22 PM, Fanghua(Joshua) Yu (notifications@github.com)
wrote:

I'd think so too. I didn't change any settings from the repository, and
used this to run:

$ python3.6 -m macgraph.train

The last lines on screen before I terminated the execution were:
... ...
INFO:tensorflow:global_step/sec: 81.7011
INFO:tensorflow:loss = 0.0, step = 10511101 (1.224 sec)
INFO:tensorflow:loss = 0.0, step = 10511101 (1.224 sec)

Where do I set up the accuracy?

Cheers

Joshua

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/Octavian-ai/gqa-node-properties/issues/1#issuecomment-469076992>,
or mute the thread
